app/models.py

```diff
@@ -550,3 +557,21 @@ def bulk_lookup():
                 db.session.commit()
             except IntegrityError:
                 db.session.rollback()
+
+class Leaders(db.Model):
```

Comment:
Thinking about it, if I use alembic to create the priority and leader views in the migration, I would still need to drop the tables first which are created with `db.create_all()`. I guess the most parsimonious solution would be to not create the models in app/models.py, and create the views in the migration. I can then reference the views using standard SQL as `leaders = db.session.execute('select * from leaders;')`. What do you think? In any case, think I'm going to put this into a separate PR.